src/Toolbar.tsx

```diff
@@ -15,23 +14,6 @@ export const sketchButtonClassnames = {
   icon: 'text-fern-20 h-auto group-hover:text-fern-10 hover:text-fern-10 dark:text-chalkboard-100 dark:group-hover:text-chalkboard-100 dark:hover:text-chalkboard-100 group-disabled:bg-chalkboard-60 hover:group-disabled:text-inherit',
 }
 
-const sketchFnLabels: Record<ToolTip | 'sketch_line' | 'move', string> = {
```

Comment:
@Irev-Dev @franknoirot was this forgotten? Or to be used later?

Reply:
Probably should have been cleaned up in the xstate migration.

Reply:
Cool, removing is the way to go then